hw/opentitan/ot_hmac.c

```diff
@@ -699,13 +983,11 @@ static void ot_hmac_regs_write(void *opaque, hwaddr addr, uint64_t value,
 
             s->regs->cmd = R_CMD_HASH_CONTINUE_MASK;
 
-            /* Restore SHA256 context */
-            s->ctx->state.sha256.curlen = 0;
-            s->ctx->state.sha256.length = s->regs->msg_length;
-            unsigned digest_length = OT_HMAC_DIGEST_LENGTH / sizeof(uint32_t);
-            for (unsigned i = 0; i < digest_length; i++) {
-                s->ctx->state.sha256.state[i] = s->regs->digest[i];
-            }
+            /* Hold the previous digest size until the HMAC is started with the
```

Comment:
Unrelated, can you take the opportunity of this change to fix the comment line width @ 968? Thanks.